index.html

```diff
@@ -1,304 +1,416 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link
-        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
-        rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&family=Poppins:wght@300&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
+  </head>
 
-<body>
-    <main class="main_content">
-        <header class="header">
-            <div class="menu-icon">
-                <a href="index.html"><img src="images/icons/logo.png" alt="Logo" /></a>
-                <nav>
-                    <img src="images/icons/Union.png" alt="Main menu">
-                </nav>
-            </div>
-        </header>
-        <section class="headline">
-            <h1 class="headline-title">
-                Hey There<br />
-                I'm Eric.
-            </h1>
-            <h2 class="headline-title2">I'm a Software Developer!</h2>
+  <body>
+    <main class="main-content">
+      <section class="header">
+        <nav class="menu-icon">
+          <h2><a href="index.html">My Logo</a></h2>
```

Comment:
- [ ] Great job with your navbar and applying the requested changes! However, one of the previous reviewers suggested aligning your "My logo" with the main headline. To achieve this, you can **add some margin to the left of your `h2` tag** inside the `navbar`. **This will make your portfolio more consistent, closer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=DOmVpbk2buQv96CX-0), and more professional.** Keep up the good work! 🎯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236648735-170149c1-7ce6-43c2-acc1-fc5ea3e5aeae.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236648752-c684329d-565d-40ee-89e0-6e32fac1f12e.png)